### PR TITLE
changelog: add notes for 0.4.3-rc4

### PR DESCRIPTION
**Pull request by lgierth**
Please review in the rendered version to spot markup errors

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # go-ipfs changelog
 
+### 0.4.3-rc4 - 2016-09-06
+
+This release candidate fixes issues in Bitswap and the `ipfs add` command, and improves testing.
+We plan for this to be the last release candidate before the release of go-ipfs v0.4.3.
+
+With this release candidate, we're also moving go-ipfs to Go 1.7, which we expect to yield improvements in runtime performance, memory usage, build time and size of the release binaries.
+
+- Fix output of commands taking input from stdin. (@Kubuxu, [ipfs/go-ipfs#3134](https://github.com/ipfs/go-ipfs/pull/3134))
```
Comment (Kubuxu) [inline CHANGELOG.md]: Fix __help__ notice

Comment (lgierth) [inline CHANGELOG.md]: Would this work? "Fix info notice of commands taking input from stdin."

To me *help* sounds like it's about the `--help` text.

Comment (Kubuxu) [inline CHANGELOG.md]: +1

Comment (lgierth) [inline CHANGELOG.md]: ok fixed, and reordered the entries roughly by category

**Comment by RichardLitt** [comment]
LGTM

**Comment by lgierth** [comment]
Updated with the last PR

**Pull request merged**